mautic/mautic
# [DPMMA-2641] Preference Center button "Cancel"

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [✅]
| New feature/enhancement? (use the a.x branch)      | [❌]
| Deprecations?                          | [❌]
| BC breaks? (use the c.x branch)        | [❌]
| Automated tests included?              | [❌] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



(Version 5) The default unsubscribe page started displaying additional fields and a "Cancel" button below the save button. PR changes the way the form ends without listing previously unincluded fields.

![cancel](https://github.com/mautic/mautic/assets/102536220/54a8280c-3842-4804-90bd-5e4a4a8d1492)
![cancel2](https://github.com/mautic/mautic/assets/102536220/be12de1a-aa5e-4ecb-818a-fde8bd36c5b1)

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Change Configuration => Email Settings to
![segments](https://github.com/mautic/mautic/assets/102536220/ee56197f-f8eb-41bb-9ab2-80b271b27dbc)
3. Create email and send to test contacts. 
4. Check the unsubscribe page. Additional fields and the "Cancel" button should not be printed. 
5. Make test changes and save the form. Check whether the changes have been saved correctly for the contact.
